scholly/base/src/layouts/layout3/components/landingPage/LandingPage.js

```diff
@@ -21,13 +22,18 @@ const landingPage = () => {
           {' '}
         </title>
       </Helmet>
-      <Container fluid className="pages-top-padding">
-        <div className="landing-center-title">About Us</div>
-        {/* eslint-disable-next-line jsx-a11y/alt-text */}
-        <img src={profilePicture} align="left" className="left" />
-        {homepageData.aboutUs.map((paragraph) => (
-          <div className="landing-center-content">{paragraph}</div>
-        ))}
+      <Container fluid className="pages-top-padding pages-side-padding">
+        <Container fluid className="container-body">
+          {/* eslint-disable-next-line jsx-a11y/alt-text */}
+          <img src={landingPicture} align="left" className="left landing-image" style={{ width: '35%' }} />
```

Comment:
Should be easy to add an `alt` tag here so that we can remove the ESLint ignore line

Reply:
Fixed. Is my value for alt tag appropriate?